js/src/collapse.js

```diff
@@ -203,36 +203,35 @@ const Collapse = (() => {
 
     hide() {
       if (this._isTransitioning ||
-        !$(this._element).hasClass(ClassName.SHOW)) {
+        !this._element.classList.contains(ClassName.SHOW)) {
         return
       }
 
-      const startEvent = $.Event(Event.HIDE)
-      $(this._element).trigger(startEvent)
-      if (startEvent.isDefaultPrevented()) {
+      const startEvent = EventHandler.trigger(this._element, Event.HIDE)
+      if (startEvent.defaultPrevented) {
         return
       }
 
-      const dimension       = this._getDimension()
+      const dimension = this._getDimension()
 
       this._element.style[dimension] = `${this._element.getBoundingClientRect()[dimension]}px`
 
       Util.reflow(this._element)
 
-      $(this._element)
-        .addClass(ClassName.COLLAPSING)
-        .removeClass(ClassName.COLLAPSE)
-        .removeClass(ClassName.SHOW)
+
+      this._element.classList.add(ClassName.COLLAPSING)
+      this._element.classList.remove(ClassName.COLLAPSE)
+      this._element.classList.remove(ClassName.SHOW)
```

Comment:
You can comma separate the selectors. Same goes for `add` BTW.

Reply:
they are no support in IE10 for that : https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#Browser_compatibility

Reply:
Ah, all right. Freaking IE.